de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/dataaccess/TransactionalJsonProviderTest.java

```diff
@@ -29,71 +30,75 @@
 import javax.xml.bind.JAXBContext;
 import javax.xml.bind.JAXBException;
 
-import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.emf.common.command.Command;
+import org.eclipse.emf.edit.command.AddCommand;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.eclipse.jgit.api.errors.NoHeadException;
+import org.eclipse.jgit.revwalk.RevCommit;
 import org.junit.Before;
 import org.junit.Test;
 
 import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyInt;
-import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyString;
-import de.dlr.sc.virsat.model.dvlm.categories.CategoriesFactory;
-import de.dlr.sc.virsat.model.dvlm.categories.Category;
-import de.dlr.sc.virsat.model.dvlm.categories.CategoryAssignment;
-import de.dlr.sc.virsat.model.dvlm.categories.propertydefinitions.PropertydefinitionsFactory;
-import de.dlr.sc.virsat.model.dvlm.categories.propertydefinitions.StringProperty;
-import de.dlr.sc.virsat.model.dvlm.categories.propertyinstances.PropertyinstancesFactory;
-import de.dlr.sc.virsat.model.dvlm.categories.propertyinstances.ValuePropertyInstance;
+import de.dlr.sc.virsat.model.concept.types.structural.BeanStructuralElementInstance;
+import de.dlr.sc.virsat.model.dvlm.DVLMPackage;
+import de.dlr.sc.virsat.model.dvlm.roles.UserRegistry;
 import de.dlr.sc.virsat.model.dvlm.structural.StructuralElement;
 import de.dlr.sc.virsat.model.dvlm.structural.StructuralElementInstance;
 import de.dlr.sc.virsat.model.dvlm.structural.StructuralFactory;
-import de.dlr.sc.virsat.project.test.AProjectTestCase;
+import de.dlr.sc.virsat.project.editingDomain.VirSatTransactionalEditingDomain;
+import de.dlr.sc.virsat.project.resources.command.CreateSeiResourceAndFileCommand;
+import de.dlr.sc.virsat.server.test.AServerRepositoryTest;
 
-public class TransactionalJsonProviderTest extends AProjectTestCase {
+public class TransactionalJsonProviderTest extends AServerRepositoryTest {
 
 	private TransactionalJsonProvider provider;
-	private BeanPropertyString testBean;
+	private BeanStructuralElementInstance testBean;
 	private Class<?> type;
 	private Set<Class<?>> beanClass = new HashSet<>();
 	private MediaType mediaType;
 	private String testString = "test";
 
 	@Before
-	public void setUp() throws CoreException {
+	public void setUp() throws Exception {
 		super.setUp();
-		addEditingDomainAndRepository();
+		
+		UserRegistry.getInstance().setSuperUser(true);
```

Comment:
Super important: We also have to set this to false after the test case is done because the super user flag is some static object so it gets carried into the next test case.

Reply:
Yes, sorry I forgot that